### PR TITLE
Fix a small bug + add 2 convenience functions

### PR DESCRIPTION
**Pull request by stelmo**
These 2 convenience functions will get changed with #700, but for now it is useful :)

### CODE REVIEW

File: src/utils/BalancedGrowthCommunityModel.jl [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+) : nothing
+
+
+"""
+$(TYPEDSIGNATURES)
+
+Extract the environmental exchanges from `opt_model`, which is a solved
+optimization model built from the `community_model`.
+"""
+get_environmental_exchanges(opt_model, community_model::BalancedGrowthCommunityModel) = is_solved(opt_model) ? Dict(
```
Comment (exaexa) [inline src/utils/BalancedGrowthCommunityModel.jl:19]: consider keeping the arguments in a consistent order, preferably sorted by "logical size" ( model ≥ optmodel ≥ piece of model ).

Other than that I'm okay with this.

**Review by exaexa — APPROVED** [review]
1 suggestion worthy considering, other than that OK

File: src/utils/BalancedGrowthCommunityModel.jl (hunk 1 above)
Comment (stelmo) [reply src/utils/BalancedGrowthCommunityModel.jl:19]: cool, will change!

**Pull request merged**